pycaret/internal/helper.py

```diff
@@ -0,0 +1,677 @@
+# report helper
+
+
+# Specific to model type
+def get_plot_list(model_name, model_type):
+    plot_list = ''
+    if model_type == 'regression':
+        plot_list = get_plot_list_regression(model_name)
+    elif model_type == 'classification':
+        plot_list = get_plot_list_classification(model_name)
+    elif model_type == 'clustering':
+        plot_list = get_plot_list_clustering(model_name)
+    elif model_type == 'anomaly_detection':
+        plot_list == get_plot_list_anomaly_detection(model_name)
+    elif model_type == 'nlp':
+        plot_list = get_plot_list_nlp(model_name)
+    return plot_list
+
+
+def get_plot_list_regression(model_name):
```

Comment:
I don't think plot lists are sustainable. I think it would be a better idea to:
- Define a list of plots that should never be created
- Try and create all plots through `plot_models` for the model - incorrect plots will raise an exception, which can be caught and ignored.

Reply:
Define a list of plots that should never be created -- done